java/pgv-artifacts/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>pgv-java</artifactId>
         <groupId>io.envoyproxy.protoc-gen-validate</groupId>
-        <version>0.4.2-SNAPSHOT</version>
+        <version>0.6.1</version>
```

Comment:
This should be 0.6.1-SNAPSHOT.
The maven release plugin will modify this when its invoked.